extensions/amp-mathml/1.0/storybook/Basic.js

```diff
@@ -0,0 +1,94 @@
+import * as Preact from '#preact';
+
+import {BentoMathml} from '../component';
+
+const formulas = {
+  quadratic: '\\[x = {-b \\pm \\sqrt{b^2-4ac} \\over 2a}.\\]',
+  cauchy: '\\[f(a) = \\frac{1}{2\\pi i} \\oint\\frac{f(z)}{z-a}dz\\]',
+  doubleAngle: '\\[cos(θ+φ)=\\cos(θ)\\cos(φ)−\\sin(θ)\\sin(φ)\\]',
+};
+
+export default {
+  title: 'Mathml',
+  component: BentoMathml,
+  args: {
+    formulas,
+  },
+};
+
+export const Default = (args) => {
+  // should not render formulas
```

Comment:
suggestion: These comments can be included in the rendered HTML so that Storybook users can see the notes with the manual samples.